src/scss/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$primary: #ff7022;
```

Comment:
Although this meets requirements, this is close to the minimum amount of customisation possible. Your design doesn't require more than one special colour? You are happy with all other design choices made in Bootstrap?

Reply:
Yes, I will look into this. See if i can add some more tweaks throughout the weekend..

Thanks for good feedback! Have a great weekend..